src/containerapp/azext_containerapp/_help.py

```diff
@@ -256,6 +264,45 @@
           az containerapp env dapr-component remove -g MyResourceGroup --dapr-component-name MyDaprComponentName --name MyEnvironment
 """
 
+# Identity Commands
+helps['containerapp identity'] = """
+    type: group
+    short-summary: Commands to manage managed identities.
+"""
+
+helps['containerapp identity assign'] = """
+    type: command
+    short-summary: Assign managed identity to a container app.
+    long-summary: Managed identities can be user-assigned or system-assigned.
+    examples:
+    - name: Assign system identity.
+      text: |
+          az containerapp identity assign
+    - name: Assign user identity.
+      text: |
+          az containerapp identity assign --identities myAssignedId
```

Comment:
NIT: just to clarify this is user-assigned can we call this 'myUserAssignedId'

Reply:
This comment is for all examples in help. There is no -n (containerapp name) in the help commands. I would expect this to be required

Reply:
Added